src/main/zip/info.xml

```diff
@@ -24,6 +28,12 @@
   </author>
 
   <!--
+<<<<<<< HEAD
+  **intergration type IS Required**
+  The integration type section identifies where the plugin fits into the process
+  excepted types are SCM, Build, Artifact, Automation, Deploy
+  For example:
+=======
```

Comment:
Unresolved merge conflicts. These are causing the plugin build to fail.

Please see the following Git documentation to help explain how to handle merge conflicts:
https://help.github.com/en/articles/resolving-a-merge-conflict-using-the-command-line